Src/Common/W3C/W3CActivityExtensions.cs

```diff
@@ -0,0 +1,210 @@
+﻿namespace Microsoft.ApplicationInsights.W3C
+{
+    using System;
+    using System.ComponentModel;
+    using System.Diagnostics;
+    using System.Globalization;
+    using System.Linq;
+
+    /// <summary>
+    /// Extends Activity to support W3C distributed tracing standard.
+    /// </summary>
+    [Obsolete("Not ready for public consumption.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Same question here - should it be applied to individual methods?

Reply:
fixed